src/main/java/marquez/client/models/Field.java

```diff
@@ -0,0 +1,37 @@
+package marquez.client.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class Field {
+  String name;
+  FieldType type;
+  @Nullable List<String> tags;
+  @Nullable String description;
+
+  @JsonCreator
+  public Field(
+      @NonNull final String name,
+      @NonNull final FieldType type,
```

Comment:
Let's make `Field.type` a string type. See [comment below](https://github.com/MarquezProject/marquez-java/pull/58/files#r374429582) for context.

Reply:
Done